source/features/patch-diff-links.tsx

```diff
@@ -1,37 +1,42 @@
 import React from 'dom-chef';
-import elementReady from 'element-ready';
 import * as pageDetect from 'github-url-detection';
 
 import features from '../feature-manager.js';
 import {getCleanPathname} from '../github-helpers/index.js';
+import observe from '../helpers/selector-observer.js';
 
-async function init(): Promise<void> {
+async function addPatchDiffLinks(commitMeta: HTMLElement): Promise<void> {
 	let commitUrl = '/' + getCleanPathname();
 
 	// Avoids a redirection
 	if (pageDetect.isPRCommit()) {
 		commitUrl = commitUrl.replace(/\/pull\/\d+\/commits/, '/commit');
 	}
 
-	const commitMeta = await elementReady('.commit-meta');
 	commitMeta!.classList.remove('no-wrap'); // #5987
-	commitMeta!.lastElementChild!.append(
-		<span className="sha-block" data-turbo="false">
-			<a href={`${commitUrl}.patch`} className="sha">patch</a>
+	commitMeta!.append(
+		<span className="sha-block ml-2" data-turbo="false">
+			<a href={`${commitUrl}.patch`} className="sha color-fg-default">patch</a>
 			{' '}
-			<a href={`${commitUrl}.diff`} className="sha">diff</a>
+			<a href={`${commitUrl}.diff`} className="sha color-fg-default">diff</a>
 		</span>,
 	);
 }
 
+async function init(signal: AbortSignal): Promise<void> {
+	observe([
+		'.commit-meta', // TODO remove in March 2025
```

Comment:
The comment wasn't updated

Reply:
It's important that we only mark code for removal if we're sure that the new version completely replaces it. 

Out of the open 3 issues (#7856 #7886 #7808), only features that run on _only_ on `isSingleIssue` are guaranteed to be removed. If a feature runs on `isConversation`, then it's here to stay because PRs are not affected for example.

Reply:
I think updated it, but it seems I discarded it.